tests/keras/backend/backend_test.py

```diff
@@ -1689,6 +1689,21 @@ def test_ctc_decode_greedy(self):
         assert np.alltrue(decode_pred_np == decode_pred)
         assert np.allclose(log_prob_pred_np, log_prob_pred)
 
+    @pytest.mark.skipif(K.backend() != 'tensorflow',
+                        reason='tensorflow-way slice is '
+                        'only supported in tensorflow.')
+    def test_slice(self):
+        npt = np.array([[[1, 1, 1], [2, 2, 2]],
+                       [[3, 3, 3], [4, 4, 4]],
+                       [[5, 5, 5], [6, 6, 6]]])
+        tft = K.constant(npt)
+        test_1 = K.eval(K.slice(tft, [1, 0, 0], [1, 1, 3]))
+        test_2 = K.eval(K.slice(tft, [1, 0, 0], [1, 2, 3]))
+        test_3 = K.eval(K.slice(tft, [1, 0, 0], [2, 1, 3]))
```

Comment:
Since you are doing three tests, you should be able to use pytest's `parametrize` feature. https://docs.pytest.org/en/latest/parametrize.html you can see many examples in the backend_test.py.